### PR TITLE
[label] Pharma, Yoshitoshi Recordings, Guerilla

### PR DESCRIPTION
**Pull request by romalozhkin00001**

### CODE REVIEW

File: discogs-labels.csv [hunk 1]
```diff
@@ -291,3 +291,10 @@ Label ID, Label Name, Link
 303, Missile Records, https://www.discogs.com/label/303-Missile-Records
 3435, Rampant, https://www.discogs.com/label/3435-Rampant
 10112, Ozone, https://www.discogs.com/label/10112-Ozone-Recordings
+18, Soma Quality Recordings, https://www.discogs.com/label/18-Soma-Quality-Recordings
```
Comment (stephanebruckert) [inline discogs-labels.csv:294]: This one is a duplicate, please remove the line

**Review by stephanebruckert — CHANGES_REQUESTED** [review]
I just enabled automated checks to look for duplicates, see https://github.com/mirrorfm/mirrorfm-data/actions/runs/3332732449/jobs/5513941548#step:4:306

File: discogs-labels.csv (hunk 1 above)
Comment (romalozhkin00001) [reply discogs-labels.csv]: done

Comment (romalozhkin00001) [reply discogs-labels.csv]: How can I find this playlist? I don't see it on your profile

Comment (stephanebruckert) [reply discogs-labels.csv]: ha yeah there is only 50 max on the spotify profile
You can find all of them on [mirror.fm](https://mirror.fm/labels/)
https://mirror.fm/discogs/18/soma-quality-recordings/
https://open.spotify.com/playlist/6ZX0GWJpulkMuOlS0NNdnC?si=6e9dcc7f622a4843

Comment (romalozhkin00001) [reply discogs-labels.csv]: Big thanks!

**Pull request merged**